tests/Makefile

```diff
@@ -5,8 +5,8 @@
 PSQL ?= TEST_USE_POSTGRESQL=true
 PG800 ?= CODECHECKER_DB_DRIVER=pg8000
 PSYCOPG2 ?= CODECHECKER_DB_DRIVER=psycopg2
-DBPORT ?= TEST_DBPORT=5432
-DBUNAME ?= TEST_DBUSERNAME=postgres
+DBPORT ?= TEST_DBPORT=4995
```

Comment:
I think these should be set back to the original values.

Reply:
@dkrupp Always use `git add --patch`. :wink: